### ISSUE TITLE
[BUG] Installation installation link is dead

### ISSUE THREAD

**Issue by davols** [Boerderij/Varken#248]
**Wiki installation link is dead**
Varken Installation guides can be found in the [wiki](https://wiki.cajun.pro/books/varken/chapter/installation).

**To Reproduce**
1. Click wiki @ [wiki](https://wiki.cajun.pro/books/varken/chapter/installation)
2. Notice it gives you an error

**Expected behavior**
Should lead to wiki. 

**Screenshots**
![2022-10-15-235850_678x88_scrot](https://user-images.githubusercontent.com/1420172/196009133-f5c328fa-b1d8-47c0-8245-5f0952dd0f3a.png)

**Additional context**
`wiki.cajun.pro` returns 404 as well. Possibly intermittent BW issues?

**Comment by Spellbinder4** [comment]
Is this project still alive?

Would love to get this link working so I can get some guidance to getting it all working.

**Comment by aelberling** [comment]
https://old.reddit.com/r/homelab/comments/a4txoe/i_introduce_varken_the_successor_of/#ix704i3
Looks like dead. Shame that he hasn't updated the repo to reflect EoL.

**Comment by MDHMatt** [comment]
> 

its not dead just slow updates. been taken over by a few people just trying to keep it going :)

**Comment by aelberling** [comment]
> > 
> 
> its not dead just slow updates. been taken over by a few people just trying to keep it going :)

Fair enough.

**Comment by M1kep** [comment]
@MDHMatt Are there backups of the setup instructions?

**Comment by MDHMatt** [comment]
> @MDHMatt Are there backups of the setup instructions?

Do you know how to use docker?? That is by far the easiest way to do it

**Comment by M1kep** [comment]
> > @MDHMatt Are there backups of the setup instructions?
> 
> Do you know how to use docker?? That is by far the easiest way to do it

I understand docker, and most likely how to deploy it, I'm personally looking for the docs on the configuration values. Screenshot taken from waybackmachine
<img width="803" alt="CleanShot 2022-12-23 at 11 57 21@2x" src="https://user-images.githubusercontent.com/2661819/209400754-7c4cc241-a1b5-4f1d-8475-c1a51ef0c651.png">

Found https://github.com/Boerderij/Varken/blob/develop/data/varken.example.ini, which should hopefully suffice

**Comment by MDHMatt** [comment]
@M1kep yeah run the develop branch and edit that ini file with the ip addresses of the apps, any api keys and the influxdb details and away you go
